scripts/etch-language-tests/ml/graph.etch

```diff
@@ -0,0 +1,24 @@
+function main()
```

Comment:
This test is missing assertions that behaviour is correct. Etch now contains an `assert` function you can use, with an optional message.